### PR TITLE
Update: updated import for marked (fixes #32)

### LINKED ISSUE #32: Carbon Site fails in IE11 after upgrade from marked@0.7.0 to 0.8.0
**Describe the bug**
A syntax error is thrown when you try to access carbon's dev site, which uses this helper. There is a known error in markedjs@0.8.0 that breaks support of IE.

Workaround specified in the markedjs github issue is to import marked from lib:
https://github.com/markedjs/marked/issues/1585#issuecomment-570830943

**To Reproduce**

1. Launch the carbon dev site in IE
https://engineering.cerner.com/carbon-graphs/#/

**Expected behavior**
The site should render in Internet Explorer

**Screenshots**
image

**Desktop (please complete the following information):**

-   OS: Windows 10
-   Browser Internet Explorer 11

### PR DESCRIPTION
**Pull request by pranav300**
**What is the purpose of this pull request? (Check any that's applicable)**

-   [ ] Feature
-   [X] Bug fix
-   [ ] Dependency updates
-   [ ] Documentation updates
-   [ ] Build related changes
-   [ ] Changes an existing functionality
-   [ ] Other, please explain:

**Does this introduce a breaking change?**

-   [ ] Yes
-   [X] No

**What changes did you make? (Give an overview)**
Updated import for package `marked`.
**Add any screenshots**

### CODE REVIEW

**Comment by swatikode** [comment]
What bug was this code causing?

File: src/pages/Releases/ReleasesPage.js [hunk 1]
```diff
@@ -19,6 +18,8 @@ import Header from "../../components/Header/Header";
 import { isLatestRelease } from "../../helpers/releasesInfo";
 import { useReleasesURL } from "../../providers/releases/ReleasesURLProvider";
 
+const marked = require('marked/lib/marked');
```
Comment (swatikode) [inline src/pages/Releases/ReleasesPage.js:21]: If there is no difference in the usage of `marked` then what issue is this PR solving?

**Comment by pranav300** [comment]
>What bug was this code causing?

This PR would solve issue #32.

File: src/pages/Releases/ReleasesPage.js (hunk 1 above)
Comment (pranav300) [reply src/pages/Releases/ReleasesPage.js:21]: I guess its the difference between import and require, which somehow works.

**Comment by swatikode** [comment]
I don't think we should even bother fixing any IE related issues. Carbon site is just a reference site. If you go to `github.com`, even it tells that they are not supporting IE anymore. There is no strong reason to fix this issue. Once you fix this, you might see yourself uncover other IE11 issues. You will come across issues with webpack not working with IE. So I recommend not to worry about fixing any IE related issues.

**Comment by pranav300** [comment]
@swatikode 
I get your point on why we shouldn't fix IE issues. But wouldn't that hinder the development and testing in carbon-graphs? Because we still support IE  in carbon-graphs.

**Comment by swatikode** [comment]
Graphs will still render in IE, it is the site that is not loading. Are you using the site as a primary site for testing? It

**Comment by swatikode** [comment]
Sorry closed by mistake.

**Comment by ShettyAkarsh** [comment]
> Graphs will still render in IE, it is the site that is not loading. Are you using the site as a primary site for testing? There have been concerns of including [non-IE10+ supported code](https://github.com/cerner/carbon-graphs/pull/138/files/de9a281efeef72daa41717c75a9f9ced2d2ece4a..8f47d2d38e7ec545ad9c1fc540aeb2ba6676510e#r383953637) in Carbon, so we might have to address that as well. The version of IE for rendering also depends on which application is using Carbon. For certain Cerner applications, IE version is forced to Edge or similar. So you may see it in working for some instances.

@swatikode , I got your point this is just a reference site as the consumer will not use this.  but still when consumers try to check our examples in different browsers using dev-site. he expects there should be zero errors.  if this error is not allowing our graphs examples to be opened in IE. then my thought is to fix it.

**Pull request merged**